test/inductor/test_cpu_repro.py

```diff
@@ -1808,7 +1808,7 @@ def get_index():
     @patch("torch.cuda.is_available", lambda: False)
     def test_maxpool2d_cpu_only(self):
         for dtype in vec_dtypes:
-            input = torch.randn(10, 32, 20, 20, dtype=dtype).to(
+            input = torch.randn(26, 32, 112, 112, dtype=dtype).to(
```

Comment:
The increase of shape sizes is necessary to trigger the failures?

Reply:
Yes, a smalled shape can't triger the eerror.